tests/js/error_tests.js

```diff
@@ -0,0 +1,34 @@
+
+export const test = async (fluvio, offset) => {
+  let base_url = "http://127.0.0.1:3000";
+  try {
+    await fetch(`${base_url}/off`, {
+      method: 'GET',
+    });
+  } catch (e) {
+    console.error(`${e.message}`);
+  }
```

Comment:
This call would fail in firefox for me but if I recall correctly, it worked in chrome.